examples/index_hxml.xml.njk

```diff
@@ -0,0 +1,14 @@
+---
+title: "Hyperview"
+permalink: "index_hxml.xml"
+list_tag: "root"
+hide_back_button: true
+list_href: "list.xml"
+show_hxml_toggle: true
+hxml_toggle_enabled: true
```

Comment:
minor/nit: I wonder if it's possible to use a single flag, and test for `!undefined` for "visibility" vs `== true/false` for "on/off" state. Also worth renaming to match the "navigator" terminology here.